docs/operating-eck/licensing.asciidoc

```diff
@@ -48,7 +48,7 @@ At the end of the trial period, the Platinum and Enterprise features operate in
 [float]
 [id="{p}-add-license"]
 == Add a license
-If you have a valid Enterprise subscription or a trial license extension, you will receive a license as a JSON file.
+If you have a valid Enterprise subscription or a trial license extension, you will receive a license as a JSON file. This file contains the ECK orchestration(Enterprise) license which enables ECK features, as well as individual Platinum licenses for your actual Elasticsearch cluster(s).
```

Comment:
That's not fully correct. The orchestration Enterprise license contains cluster-level Enterprise licenses as well as Platinum licenses for older Elasticsearch version.s

Reply:
Ok thanks for the note, I'll get that updated